lib/config.ts

```diff
@@ -9,6 +9,8 @@ class Config {
   readonly kafkaInstanceCreationTimeout: number;
   readonly kafkaInstanceDeletionTimeout: number;
 
+  readonly kafkaTopicDeletionTimeout: number;
```

Comment:
this is not used now, either remove or update the code to make use of it.

Reply:
removed